src/main/java/com/amazon/dlic/auth/http/jwt/HTTPJwtAuthenticator.java

```diff
@@ -84,10 +84,6 @@ public HTTPJwtAuthenticator(final Settings settings, final Path configPath) {
         if (jwtParserBuilder == null) {
             jwtParser = null;
         } else {
-            if (requireAudience != null) {
-                jwtParserBuilder.requireAudience(requireAudience);
```

Comment:
I was contemplating making a change in https://github.com/jwtk/jjwt for the ParserBuilder to support this use-case, but have not submitted a PR yet. It would be nice for this to be supported in the library that the security plugin relies on, but this solution looks good in the interim!

Reply:
> I was contemplating making a change in https://github.com/jwtk/jjwt for the ParserBuilder to support this use-case 

I looked at jjwt's code if jjwt had this feature, but it wasn't there. (java-jwt is supporting it by this [commit](https://github.com/auth0/java-jwt/pull/472))
so I thought about how to do it and ended up doing it like this :)

If you add this feature to jjwt, It will be nicer.